tap_dynamics/client.py

```diff
@@ -258,3 +259,11 @@ def build_params(orderby_key: str = 'modifiedon',
             filter_param = f'{replication_key} ge {filter_value}'
             return {"$orderby": orderby_param, "$filter": filter_param}
         return {"$orderby": orderby_param}
+
+    
+    @staticmethod
+    def build_select_params(desired_columns: list):
+        select_columns = ','.join(desired_columns)
+        if len(select_columns) > MAX_SELECT_PARAM_SIZE or len(desired_columns) == 0:
```

Comment:
is there size limitation from ms dynamics side?

Reply:
I couldn't find exact limitation on ms dynamics, but instead used limit on url length that most browser would accept. Most browser seems to accept 2048 chars, so left a some room for other params and url length. 
https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers

Reply:
Do we return error or we select everything when its greater than limit?

Reply:
When it's greater than limit, select everything and transformer filters the records.